extensions/wikia/Lightbox/Lightbox.i18n.php

```diff
@@ -73,11 +73,7 @@
 	'lightbox-share-email-label' => 'Email this image to a friend',
 	'lightbox-share-email-page-label' => 'Email this page to a friend',
 	'lightbox-share-email-page-label-address' => 'Email address',
-	'lightbox-share-email-ok-caption' => 'Confirmation',
 	'lightbox-share-email-ok-content' => '{{PLURAL:$1|Email has|Emails have}} been sent.',
```

Comment:
Regarding first comment: https://github.com/Wikia/app/blob/dev/extensions/wikia/Lightbox/LightboxController.class.php#L367

Regarding second comment: 
I checked the code and couldn't find any usages. I didn't won't to assume this messages keys are somehow concatenated somewhere in the code. I thought we can remove those keys and test lightbox sharing if all messages are displayed correctly. If something is broken, it is fairly easy to bring messages back, on the other hand if everything works fine we leave clean code behind us :)